presto-orc/src/main/java/com/facebook/presto/orc/reader/LongDictionarySelectiveStreamReader.java

```diff
@@ -383,7 +379,7 @@ public void close()
     public long getRetainedSizeInBytes()
     {
         return INSTANCE_SIZE +
-                sizeOf(dictionary) +
+                dictionaryMemoryUsage +
```

Comment:
If I understand correctly, this could be `getDictionaryResult.getIsNewBuffer() ? sizeOf(dictionary) : 0`